src/network/chains.ts

```diff
@@ -0,0 +1,23 @@
+import { createPublicClient, http } from 'viem';
+import { type Chain, base, baseSepolia, mainnet, sepolia } from 'viem/chains';
+
+export const baseChainsIds: Chain['id'][] = [base.id, baseSepolia.id];
+export const ensUniversalResolverChainIds: Chain['id'][] = [
+  mainnet.id,
+  sepolia.id,
+  ...baseChainsIds,
+];
+
+export const chainsById: Record<number, Chain> = {
+  [mainnet.id]: mainnet,
+  [sepolia.id]: sepolia,
+  [base.id]: base,
+  [baseSepolia.id]: baseSepolia,
+};
+
+export function getChainPublicClient(chainId: number) {
```

Comment:
Interesting, so this could actually replace our `publicClient` network utility, as this is more generic.

and the file should be called `getChainPublicClient`

Reply:
Yes, although it's just a pretty dumb shortcut right now.